### PR TITLE
Scalafmt

### PR DESCRIPTION
**Pull request by antagonist112358**
Updated '.scalafmt.conf', CI to include scalafmt check, and ran format on existing code

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -1,7 +1,31 @@
 version = 3.7.4
+
 runner.dialect = scala213
-maxColumn = 120
-docstrings.style = Asterisk
-trailingCommas = preserve
+
+align.preset = most
+align.multiline = true
+maxColumn = 160
```
Comment (reid-spencer) [inline .scalafmt.conf:7]: I would prefer we leave it at 120

File: bounded-context/src/main/scala/com/example/boundedContext/Main.scala [hunk 2]
```diff
@@ -31,4 +31,5 @@ object Main {
     log.info("starting the Kalix service")
     createKalix().start()
   }
+
```
Comment (reid-spencer) [inline bounded-context/src/main/scala/com/example/boundedContext/Main.scala:34]: Unnecessary blank line?

**Review by reid-spencer — CHANGES_REQUESTED** [review]
It looks like the bulk of the changes are a result of the change to the scalafmt.conf

The only nit I have with this is with the width of the lines. I find 160 columns too much upon review. When looking at a diff in IntelliJ IDEA, that's 320 columns side by side and that doesn't work well on my HUGE 4K screens.  IT definitely doesn't work on a laptop screen.  So, let's move that back to 120 and then I'm good with it.

File: bounded-context/src/main/scala/com/example/boundedContext/Main.scala (hunk 2 above)
Comment (reid-spencer) [reply bounded-context/src/main/scala/com/example/boundedContext/Main.scala:34]: Seems to be one of the settings in leaving blank lines after function definitions in .scalafmt.conf .. this isn't a big deal so you can ignore.

**Comment by antagonist112358** [comment]
@reid-spencer  - additional changes since last commit are just from re-running `scalafmt` and `scalafmtSbt`.

**Review by reid-spencer — APPROVED** [review]
LGTM

**Pull request merged**